pypowsybl/network/impl/pandapower_converter.py

```diff
@@ -0,0 +1,195 @@
+
+import math
+from importlib import util
+from typing import Dict
+
+import numpy as np
+import pandas as pd
+import pypowsybl._pypowsybl as _pp
+from pandas import Series
+
+from .network import Network
+from .network_creation_util import create_empty
+
+
+def convert_from_pandapower(n_pdp) -> Network:
+    if util.find_spec("pandapower") is None:
+        raise _pp.PyPowsyblError("pandapower is not installed")
+    else:
+        n = create_empty(n_pdp.name if n_pdp.name else 'network')
+
+        # create one giant substation
+        n.create_substations(id='s')
```

Comment:
creating one giant substation will have negative affects on single line diagram from pypowsybl, however here         n.create_voltage_levels(id=vl_id, substation_id=substation_id, topology_kind=topology_kind, nominal_v=nominal_v, substation_id is used but substation is not created?

Reply:
please explain!

Reply:
we cannot create on substation per voltage level as transformer in IIDM are expected to have all its legs in the same substation (otherwise we get an error).
In ieee, matpower importer (java side), we have a smart algo to automatically find a correct substation perimeter taking into account all IIDM constaints. We should expose it to python and use it here.
In the meantime, for a very basic 0.1 version of the importer it should be ok to keep the giant substation even if not ideal.